### DISCUSSION TITLE
The in-game leaderboard shakes when you are in first place

### DISCUSSION THREAD

**Discussion by Gabixel** [ppy/osu#20435] [Q&A] [answered]
As per title.
I first noticed it while playing. It can be annoying even if you only focus on the playfield. Here's a gif:

![leaderboard](https://user-images.githubusercontent.com/43073074/192033102-ec0a9eaf-ab30-4ee7-81b8-7731c3c13e44.gif)

**Comment by Gabixel** [comment]
And um... should I ping @arflyte for the accuracy and combo "overlap"?

**Comment by peppy** [comment]
Reasoning is the same as what was fixed in recent changes for the other end of the flow. Should be a matter of adding some padding.

**Comment by smoogipoo** [comment, marked as answer]
https://github.com/ppy/osu/issues/20505
